packages/contracts-bedrock/package.json

```diff
@@ -17,7 +17,7 @@
     "bindings": "cd ../../op-bindings && make",
     "build:forge": "forge build",
     "build:with-metadata": "FOUNDRY_PROFILE=echidna yarn build:forge",
```

Comment:
We can prob get rid of some deps now, the ethereum-js stuff that was only used in diff-testing.ts

Reply:
Sure- want to do that in this PR or a separate one?

Reply:
Either is fine! Prob could just add to this one

Reply:
Done :+1: 

Removed `@ethereumjs/trie` & `@ethereumjs/util`